internal/bminventory/inventory_test.go

```diff
@@ -8938,9 +8938,9 @@ var _ = Describe("UpdateInfraEnv - Ignition", func() {
 		Expect(response).To(BeAssignableToTypeOf(common.NewApiError(http.StatusBadRequest, errors.Errorf("error"))))
 	})
 
-	It("returns bad request when provided an old version", func() {
+	It("returns bad request when provided too recent version", func() {
```

Comment:
Currently, the test validates that an override with an old ignition version will return a bad request, what is the purpose of this change?

Reply:
What we do currently is we require a strict match for the ignition override, i.e. we hardcoded 3.1.0 in Assisted and this is the only override that works. So indeed, override with 3.0.0 is a non-working combination.

However, `ignition` library provides also a function "parse/merge with backwards-compatibility" which will merge any compatible version, so in such a scenario it can merge a template with 3.2.0 with override in 3.0.0 and result in a correct 3.2.0; for this reason this test is no longer valid.